src/main/java/com/google/api/gax/grpc/InstantiatingChannelProvider.java

```diff
@@ -191,9 +191,15 @@ static String getGaxVersion() {
   }
 
   private static String getGrpcVersion() {
-    String grpcVersion = ManagedChannel.class.getPackage().getImplementationVersion();
-    if (grpcVersion == null) {
-      grpcVersion = DEFAULT_VERSION;
+    String grpcVersion = DEFAULT_VERSION;
+    Properties gaxProperties = new Properties();
+    try {
+      gaxProperties.load(
```

Comment:
Maybe we should load this once for both grpc version and gax version, rather than loading from gax.properties every time?